adapters/adnuntius/adnuntius.go

```diff
@@ -202,6 +200,13 @@ func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters
 			})
 	}
 
+	endpoint, err := makeEndpointUrl(ortbRequest, a, noCookies)
```

Comment:
As of this version of the code, `noCookies` doesn't seem to get modified anywhere else. Can we discard it altogether?
```
159   func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters.RequestData, []error) {
160       var requestData []*adapters.RequestData
161       networkAdunitMap := make(map[string][]adnAdunit)
162       headers := setHeaders(ortbRequest)
163 -     var noCookies bool = false
164
165       for _, imp := range ortbRequest.Imp {
166           if imp.Banner == nil {
167               return nil, []error{&errortypes.BadInput{
168                   Message: fmt.Sprintf("ignoring imp id=%s, Adnuntius supports only Banner", imp.ID),
169               }}
170           }
*-- 11 lines: 171           var bidderExt adapters.ExtImpBidder--------------------------------------------------------------
182               }}
183           }
184
185 -         if adnuntiusExt.NoCookies == true {
186 -             noCookies = true
187 -         }
188
189           network := defaultNetwork
190           if adnuntiusExt.Network != "" {
*--  8 lines: 191               network = adnuntiusExt.Network---------------------------------------------------------------
199                   Dimensions: getImpSizes(imp),
200               })
201       }
202
203 -     endpoint, err := makeEndpointUrl(ortbRequest, a, noCookies)
    +     endpoint, err := makeEndpointUrl(ortbRequest, a, adnuntiusExt.NoCookies)
204       if err != nil {
205           return nil, []error{&errortypes.BadInput{
206               Message: fmt.Sprintf("failed to parse URL: %s", err),
207           }}
208       }
adapters/adnuntius/adnuntius.go
```

Reply:
The idea here is that the `noCookies` is set as per imp. that means that there might be 3 imps that where only one have the `noCookies` option. that means that the entire request to the endpoint would need the parameter set in the url. That's kind of why I opted to set a variable before the for loop so that if there's one we set it for all.

If that makes sense :D

Reply:
Ohhh I missed it is outside of the `for` loop. It does make sense, my bad. Yep, let's leaving as is.